### ISSUE TITLE
BottomSheetDragHandleView cannot be used in Sketchware Pro 

### ISSUE THREAD

**Issue by TanTai97o** [Sketchware-Pro/Sketchware-Pro#1085]
**Describe the bug**

```
java.lang.NoSuchMethodError: No virtual method setAccessibilityDelegateView(Landroid/view/View;)V in class Lcom/google/android/material/bottomsheet/BottomSheetBehavior; or its super classes (declaration of 'com.google.android.material.bottomsheet.BottomSheetBehavior' appears in /data/app/~~g_UUTyODu1XWCdBbUohfMw==/com.test.bottomsheet.ui-JchiWTvGIOQaFLKuP1x95w==/base.apk!classes37.dex)
	at com.google.android.material.bottomsheet.BottomSheetDragHandleView.setBottomSheetBehavior(BottomSheetDragHandleView.java:146)
	at com.google.android.material.bottomsheet.BottomSheetDragHandleView.onAttachedToWindow(BottomSheetDragHandleView.java:117)
	at android.view.View.dispatchAttachedToWindow(View.java:22479)
	at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3695)
	at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3695)
	at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3695)
	at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3695)
	at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3695)
	at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3695)
	at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3695)
	at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3695)
	at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3695)
	at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3695)
	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:3669)
	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:3116)
	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:10885)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1301)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1309)
	at android.view.Choreographer.doCallbacks(Choreographer.java:923)
	at android.view.Choreographer.doFrame(Choreographer.java:852)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1283)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:226)
	at android.os.Looper.loop(Looper.java:313)
	at android.app.ActivityThread.main(ActivityThread.java:8762)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:604)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067)

```

Device and Sketchware Pro-related info:
 - Device: SamSungA51
 - OS:  Android 13
 - Sketchware Pro version:v6.4.0-SNAPSHOT-fd8a064-minApi26

**Comment by remaker17** [comment]
`BottomSheetDragHandleView` requires Material3 theme in your application.

**Comment by TanTai97o** [comment]
> `BottomSheetDragHandleView` requires Material3 theme in your application.

I'm Using Material3 And Can't Use It

**Comment by remaker17** [comment]
> > `BottomSheetDragHandleView` requires Material3 theme in your application.
> 
> I'm Using Material3 Theme And Can't Use It

Can you send your code here?

**Comment by TanTai97o** [comment]
> > > `BottomSheetDragHandleView` requires Material3 theme in your application.
> > 
> > I'm Using Material3 Theme And Can't Use It
> 
> Can you send your code here?

Code Xml or Java?

**Comment by remaker17** [comment]
> > > > `BottomSheetDragHandleView` requires Material3 theme in your application.
> > > 
> > > I'm Using Material3 Theme And Can't Use It
> > 
> > Can you send your code here?
> 
> Code Xml or Java?

Both Java and XMl, paste your code in pastebin and send here link.

**Comment by TanTai97o** [comment]
> > > > > `BottomSheetDragHandleView` requires Material3 theme in your application.
> > > > 
> > > > I'm Using Material3 Theme And Can't Use It
> > > 
> > > Can you send your code here?
> > 
> > Code Xml or Java?
> 
> Both Java and XML, paste your code in pastebin and send here link.

https://pastebin.com/SWkG0C4G

**Comment by hasrat-ali** [comment]
The built-in material library and their dependencies are just updated, you can use it without any error.

**Comment by TanTai97o** [comment]
> The built-in material library and their dependencies are just updated, you can use it without any error.

Package parsing error

![Screenshot_20241123_123854_Package installer.jpg](https://github.com/user-attachments/assets/388656a7-7e18-4ea1-8fa6-212a51c85111)

**Comment by hasrat-ali** [comment]
There is still problem in signing apk.
Idk why the hell @ilyassesalama merged #1353

**Comment by hasrat-ali** [comment]
For now decrease the targetSdkVersion to 28 and then build apk again. And don't forget to reset `exculded library`, and local library if there's any.

**Comment by TanTai97o** [comment]
> For now decrease the targetSdkVersion to 28 and then build apk again. And don't forget to reset `exculded library`, and local library if there's any.

The package parsing problem still occurs.

**Comment by ilyassesalama** [comment]
> There is still problem in signing apk. Idk why the hell @ilyassesalama merged #1353

I tested it and it works fine. I believe this is an Android-version-specific issue.
